code/TestAutomation.Tests.BDD/Features/EpamSiteCareersPageTestsBDT.feature

```diff
@@ -0,0 +1,30 @@
+﻿Feature:EpamSiteCareersPageTestsBDT
+
+As a user interested in the Epam company
+I want to be able to reviw Careers page
+In order to be able to get information about opend roles in the Epam company
+
+
+@Smoke
+@CareersLokations
+Scenario: Epam Site - Careers Page - Locations
```

Comment:
Please do not forget to add purpose of the scenario that starts from word Check
Ex: Check Locations presence

Reply:
done